muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/coursepicker/CourseSignupDetailsWorkspace.java

```diff
@@ -0,0 +1,55 @@
+package fi.otavanopisto.muikku.plugins.coursepicker;
+
+
+public class CourseSignupDetailsWorkspace {
+
+  public CourseSignupDetailsWorkspace() {
+  }
+
+  public CourseSignupDetailsWorkspace(Long id, String description, boolean canSignup, boolean isCourseMember) {
```

Comment:
This class could be renamed `WorkspaceSignupDetailsRestModel`. We've never had strict naming conventions for REST objects but we really should. Whenever new stuff is introduced, I prefer to slap `RestModel` after the class name of any REST object so that when you're later wondering whether some class is an entity or just an utility object or whatever, the class name immediately tells you that yes, I'm a REST object that bounces back and forth between the frontend and the backend.